src/appleseed/renderer/meta/tests/test_projectfilewriter.cpp

```diff
@@ -60,301 +62,230 @@ TEST_SUITE(Renderer_Modeling_Project_ProjectFileWriter)
 {
     struct Fixture
     {
-        const filesystem::path      m_base_output;
-        const filesystem::path      m_alternate_output;
+        const path                  m_input_directory;
+        const path                  m_output_directory;
         auto_release_ptr<Project>   m_project;
 
         Fixture()
-          : m_base_output(filesystem::absolute("unit tests/outputs/test_projectfilewriter/"))
-          , m_alternate_output(filesystem::absolute("unit tests/outputs/test_projectfilewriter/alternate/"))
+          : m_input_directory(canonical("unit tests/inputs/test_projectfilewriter/"))
+          , m_output_directory(canonical("unit tests/outputs/test_projectfilewriter/"))
         {
-            filesystem::remove_all(m_base_output);
-            filesystem::remove_all(m_alternate_output);
+            remove_all(m_output_directory);
 
-            filesystem::create_directory(m_base_output);
-            filesystem::create_directory(m_alternate_output);
+            // On Windows, the create_directory() call below will fail with an Access Denied error
+            // if a File Explorer window was opened in the output directory that we just deleted.
+            // A small pause solves the problem.
+            sleep(50);
```

Comment:
Right, it probably needs namespace qualification. I'll push a fix right away.

Interestingly, the Travis build has succeeded.